error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -264,6 +264,32 @@ Flux<T> after(Flux<T> flux, long n) {
     }
   }
 
+  /** Prefer {@link Mono#defaultIfEmpty(Object)} over more contrived alternatives. */
+  static final class MonoSwitchIfEmptyOfMonoJust<T> {
```

Comment:
Since we name the rules after the preferred expression: 
```suggestion
  static final class MonoDefaultIfEmpty<T> {
```